models/stg_twitter_ads__line_item_keywords_report.sql

```diff
@@ -22,6 +22,7 @@ final as (
     
     select 
         {{ dbt_utils.date_trunc('day', 'date') }} as date_day,
+        {{ dbt_utils.surrogate_key(['account_id', 'line_item_id','segment','placement']) }} as keyword_id,
```

Comment:
in the yml, we should probably change the model's test to reference `keyword_id` instead of the combination of columns (also gotta add `keyword_id` to the yml docs anyways)

Reply:
will need to rerun docs after adding the column